frontend/src/metabase/containers/Form.jsx

```diff
@@ -227,7 +227,10 @@ class Form extends React.Component {
         const errorNames = Object.keys(error.data.errors);
         const hasUnknownFields = errorNames.some(name => !fieldNames.has(name));
         throw {
-          _error: hasUnknownFields ? t`An error occurred` : null,
+          _error:
+            error.data?.message ||
+            error.message ||
+            (hasUnknownFields ? t`An error occurred` : null),
```

Comment:
I couldn't use the same condition exactly as in the `else` clause because it injected `error.data` which is an object as an `error` prop which will then be shown, but objects isn't a valid React Node.

@losrebellos [you added those lines](https://github.com/metabase/metabase/pull/21668/files#r850678872). Can you help confirm if I should remove the `error.data` from the `else` clause as well?